examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
```

Comment:
`Body` implements read. You should use `read_to_end` directly, making this entire function 3 lines.

```rust
let mut data = vec![];
body.read_to_end(&mut data);
data
```